moduleframework/mtf_scheduler.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Tool to start MTF tests
+# Author Petr Sklenar psklenar@gmail.com
+#
+# Meta test family (MTF) is a tool to test components of a modular Fedora:
+# https://docs.pagure.org/modularity/
+# Copyright (C) 2017 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import argparse
+import os
+import moduleframework
+import tempfile
+import textwrap
+import sys
+
+from avocado.utils import process
+from moduleframework import common
+
+
+def cli():
+    file_name = os.path.basename(sys.argv[0])
+    parser = argparse.ArgumentParser(
+        # TODO
+        description=textwrap.dedent('''\
+        unknown arguments are forwarded to avocado:
+           optionally use additional avocado param like --show-job-log, see avocado action --help'''),
+        formatter_class=argparse.RawTextHelpFormatter,
+        # epilog(with http link) is used in some error msg too:
+        epilog="see http://meta-test-family.readthedocs.io for more info",
+        usage=("{0} [options] local_tests".format(file_name)),
+    )
+    parser.add_argument("--linter", "-l", action="store_true", default=False, help='adds additional compose checks')
+    parser.add_argument("--setup", action="store_true", default=False, help='Setup by mtfenvset')
+    parser.add_argument("--action", action="store", default='run', help='Action for avocado, see avocado --help for subcommands')
+
+    # some useful bash variables
+    # there are more possible variables up to the doc, not sure what could be like options too
+    group = parser.add_argument_group('additional arguments are like environment variables up to the http://meta-test-family.readthedocs.io/en/latest/user_guide/environment_variables.html ')
+    group.add_argument("--module", action="store", help='Module type, like: docker, nspawn or rpm')
+    group.add_argument("--debug", action="store_true", help='passes AVOCADO_LOG_DEBUG=yes DEBUG=yes')
```

Comment:
Please improve help messages, not just test that it set these envvars, some high level function of this option

Reply:
done